package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "enable-contracts",
-  "version": "1.0.0",
+  "name": "@enabledao/enable-contracts",
+  "version": "0.1.0",
   "description": "Borderless peer-to-peer loans with social attestation",
   "main": "truffle-config.js",
   "scripts": {
-    "test": "zos push --network development && truffle test",
+    "test": "zos publish --network development && zos push --network development && truffle test",
```

Comment:
Won't this create a problem where we create a new `zos.129318923.json` every time we test?

Reply:
that's a valid point, we should remove that

Reply:
If it's already published to the current network, it won't publish again

Reply:
Down to remove though